contracts/StrategyManager__v2.sol

```diff
@@ -10,9 +12,15 @@ import {ReferralStorage} from "./libraries/ReferralStorage.sol";
 
 // TODO test upgrade compatibility
 contract StrategyManager__v2 is StrategyManager {
+    using SafeERC20Upgradeable for IERC20Upgradeable;
+
     event Referral(address referrer, address referred);
+    event ReferrerPercentageUpdated(uint32 percentage);
+    event CollectedReferrerRewards(address referrer, uint amount);
 
-    // TODO add reinitializer for referrerPercentage
+    function initialize__V2(uint32 _referrerPercentage) external reinitializer(2) {
```

Comment:
```suggestion
    function initialize__v2(uint32 _referrerPercentage) external reinitializer(2) {
```

Reply:
also can this be used with `onlyOwner` (must add test calling `upgradeToAndCall` instead of just `upgradeTo`)?